GamaGameHub/GamaGameHub/Views/Shared/_Layout.cshtml

```diff
@@ -31,7 +31,7 @@
 
             <div class="s-header__logo">
                 <a class="logo" asp-area="" asp-controller="Home" asp-action="Index">
-                    <img src="~/images/logo.svg" alt="Homepage">
+                    <img src="~/images/logo.svg" alt="Homepage" style="width:150px; height:150px">
```

Comment:
The styles would better reside in the site.css and not be inlined. Please move the style there and add an appropriate name.